resources/js/components/Sorting.vue

```diff
@@ -0,0 +1,72 @@
+<script>
+export default {
+    name: 'Sorting',
+    props: {
+        inputValue: {
+            type: String,
+            default: () => ''
+        },
+        isSelect: {
```

Comment:
i also recommend to give more descriptive names for properties because at first glance its hard to understand what are they doing, so instead of `isSelect` i would name it `displaySorting` and instead of `isInput` `displaySearch` or `displayFilter`